packages/golden-layout/scss/goldenlayout-dark-theme.scss

```diff
@@ -70,7 +70,6 @@ body:not(.lm_dragging) .lm_header .lm_tab .lm_close_tab:hover {
 // Single Pane content (area in which final dragged content is contained)
 .lm_content {
   background: $content-bg;
-  overflow: visible;
```

Comment:
Looks like this is the only non-prettier change to this file. Why does this need to be removed?

Reply:
Yea it's not really related to _this_ change so I'll back it out... basically two issues I have with this:
1. Specifically I had an overflowing panel, and that was causing the entire UI to get scroll bars
2. This is the `dark-theme.scss` file, and should really only contain stuff for a "dark" theme, whereas this is about the layout of things, which is relevant in the base theme. This kind of scss should just be in the base theme instead (if it's what we actually want), instead of being overridden in the dark theme.

Created a ticket to follow up on this.